applications/TuxedoDapp.md

```diff
@@ -0,0 +1,237 @@
+
+Our goal is to showcase the potential of Polkadot using the UTXO paradigm for the end user, leveraging Tuxedo. The project will consist of two key stages:
+
+- Stage 1: Developing a simple Web Wallet
+- Stage 2: Developing a basic DApp
+
+### Project Details
+
+- Stage 1: Developing a simple Web Wallet
+
+Develop a user-friendly web wallet that allows users to manage their UTXOs on the Tuxedo-based Polkadot network which is similar to polkadot.js which will get the chain info(genesisHash, runtime metadata, chain runtime version, API version), block info
+Focus on simplicity, security, and ease of use to provide a seamless experience for users interacting with UTXOs.
+
+**Technical Requirements** 
+ 
+**Security:** : Private key storage:
+Web-based wallet will provide a secure method for private key storage on the client side, such as a browser's secure storage or hardware wallets.
+
+**Functionality**: 
+1. Feature Parity Between Web-Based and CLI Wallets
+The web-based wallet will be designed to mirror the core functionality of the CLI wallet, offering seamless UTXO synchronization and token handling, ensuring that users have an equivalent experience across both interfaces
+Note : Feature Parity: Means web-based and command-line interface (CLI) wallets, have the same set of features and functionalities and both versions provide an equivalent or identical user experience by offering the same core capabilities, options.
+
+3. Issues Fix in web-based wallet (https://github.com/Off-Narrative-Labs/Tuxedo/issues/62)
+The current implementation of a cli based wallet allows sending inputs from multiple owners in a single transaction, but it only supports a single recipient for all specified outputs.
+
+4. Browser compatibility:
+Web-based wallet will ensure compatibility with Chrome, Firefox and Edge.
+
+**Blockchain Integration**:
+Web-based wallet will connect to the blockchain node to fetch relevant data.
+We will check if there is any lib such as the one used by "https://polkadot.js.org/" to interact with the Tuxedo based blockchain for easy integration. 
+
+**Transactions**:
+web-based wallet will implement transaction creation and signing on the client side.
+We will consider using libraries that support the specific blockchain's transaction format for Tuxedo with UTXO instead of accounts.
+
+**Local Database**:
+web-based wallet will consider using databases for storing non-sensitive user data.
+Local database (e.g., IndexedDB, WebSQL, or a lightweight client-side database) to store relevant wallet data locally on the user's device. 
+
+**Technology Stack**:
+polkadot{.js}  - We will try to re-use as much as possible 
```

Comment:
Just a heads up that you may not be able to directly re-use much of this. The cryptography stuff, definitely yes, but any metadata, pallet, types related stuff most likely not.

Reply:
Yes, Thanks. We understand this. just for connecting to endpoint, fetching the block using block number or hash and anything which is not FRAME specific we will try to refer.